app/controllers/participants_controller.rb

```diff
@@ -3,8 +3,9 @@ class ParticipantsController < ApplicationController
 
   def show
     @participant = authorize Participant.find(params[:id])
-    @paperworks = @participant.paperworks
-    @case_notes = @participant.case_notes
+    @paperworks = authorize @participant.paperworks
+    @case_notes = authorize @participant.case_notes
+    @studio_assessments = authorize @participant.studio_assessments
 
     if @participant.personal_questionnaire.nil?
       personal_q = PersonalQuestionnaire.create("participant_id": @participant.id)
```

Comment:
We should authorize access to the questionnaires also right? (both personal and professional)